src/LiveComponent/doc/index.rst

```diff
@@ -2302,6 +2302,131 @@ You can also trigger a specific "action" instead of a normal re-render:
         #}
     >
 
+Binding LiveProp to URL query parameters
+----------------------------------------
+
+.. versionadded:: 2.13
+
+    The ``url`` option was introduced in Live Components 2.13.
+
+You can bind your component LiveProp's values to URL query parameters thanks to the ``url`` option::
+
+    // src/Components/SearchModule.php
+    namespace App\Components;
+
+    use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+    use Symfony\UX\LiveComponent\Attribute\LiveProp;
+    use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+    #[AsLiveComponent]
+    class SearchModule
+    {
+        use DefaultActionTrait;
+
+        #[LiveProp(writable: true, url: true)]
+        public string $query = '';
+    }
+
+Now if you change the value of the ``query`` prop, the url will be updated to reflect the new state of your
+component, for example: ``https://my.domain/search?query=my+search+string``.
+
+Then, if you load this URL in your browser, your component will be initialized with the values provided in the query
+string.
+
+.. note::
+
+    URL is updated in place, no new entry is added to the browser's history.
+
+.. warning::
+
+    You can use multiple components with URL bindings in the same page, as long as bound field names don't collide.
+    Otherwise, you will observe unexpected behaviors.
```

Comment:
Someone may have suggested adding this, I say remove it. I think this is self-evident (e.g. if you have 2 variables defined in the same function with the same name, unexpected things will happen).

Reply:
Do you suggest removing the full warning block or just the last line?